examples/ae_examples/cvae_examples/client.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+from pathlib import Path
+from typing import Sequence, Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+import torchvision.transforms as transforms
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from examples.ae_examples.cvae_examples.models import MnistConditionalDecoder, MnistConditionalEncoder
+from fl4health.clients.basic_client import BasicClient
+from fl4health.model_bases.autoencoders_base import AutoEncoderType, ConditionalVAE
+from fl4health.preprocessing.autoencoders.loss import VAE_loss
+from fl4health.preprocessing.autoencoders.vae_training import AETransformer
+from fl4health.utils.load_data import load_mnist_data
+from fl4health.utils.metrics import Metric
+from fl4health.utils.random import set_all_random_seeds
+from fl4health.utils.sampler import DirichletLabelBasedSampler
+
+
+class CondAutoEncoderClient(BasicClient):
+    def __init__(self, data_path: Path, metrics: Sequence[Metric], DEVICE: torch.device, condition: str) -> None:
+        BasicClient.__init__(self, data_path, metrics, DEVICE)
+        self.condition = condition
+
+    def get_data_loaders(self, config: Config) -> Tuple[DataLoader, DataLoader]:
+        batch_size = self.narrow_config_type(config, "batch_size", int)
+        sampler = DirichletLabelBasedSampler(list(range(10)), sample_percentage=0.75, beta=100)
+        transform = transforms.Compose([transforms.ToTensor(), transforms.Lambda(torch.flatten)])
+        # To train an autoencoder-based model we need to set the data_target_transform.
+        train_loader, val_loader, _ = load_mnist_data(
+            data_dir=self.data_path,
+            batch_size=batch_size,
+            sampler=sampler,
+            transform=transform,
+            data_target_transform=AETransformer(self.condition),
```

Comment:
The acronym naming convention that we've settled on would result in `AETransformer` -> `AeTransformer`